src/components/index.scss

```diff
@@ -1,14 +1,15 @@
-@import "icon";
-@import "grid";
+@import "Icon";
+@import "Grid";
 @import "Button";
 @import "ButtonGroup";
 @import "Checkbox";
 @import "Toggle";
 @import "Input";
 @import "SelectBox";
-@import "step-nav";
+@import "StepNav";
```

Comment:
Update directory name?

Reply:
not sure I follow on this one?  which directory?

Reply:
Oops, I meant CSS file name from step-nav to StepNav